concordium-consensus/tests/scheduler/SchedulerTests/Helpers.hs

```diff
@@ -69,6 +70,25 @@ simpleTransferCostWithMemo2 memoSize =
         1
         + Cost.simpleTransferCost
 
+-- |Generate an account with a single credential and single keypair, which has sufficiently
+-- late expiry date, but is otherwise not well-formed.
+makeTestAccount ::
```

Comment:
Would it be possible to have a version of this generating the key and address for you?

Reply:
I'll add helpers for this